### DISCUSSION TITLE
When using snowwhite with box-shadows, tags should have a box-shadow, too

### DISCUSSION THREAD

**Discussion by BurningTreeC** [TiddlyWiki/TiddlyWiki5#5927] [Ideas]
It looks very good when tags have the same box-shadow as tiddler frames
So I propose that tags get the same box-shadow when the selected theme is snowwhite

**Comment by BurningTreeC** [comment]
![screenshot_02-08-2021_075451](https://user-images.githubusercontent.com/9407182/127810990-8e89c3bc-45ce-4a7d-a674-045279b95e32.png)
![screenshot_02-08-2021_075536](https://user-images.githubusercontent.com/9407182/127810998-f4f67697-8a83-46d7-b7ec-c2ffce69588b.png)

**Reply by Jermolene** [reply]
Hi @BurningTreeC this is a fun area for experimentation. A lot of the visual design of TW5 is quite old, from a time when everything was much simpler, and it's certainly worth keeping it all under review.

In the case of the tag pills, I experimented with lots of variants, but the things I remember are:

* I felt that drop shadows took the tags out of the tiddler, making them float above it. In turn, I felt that undermined the important drop shadow: the one that makes tiddlers look like cards hovering over the page
* I tried embossed text in the tag pull, which I liked, and I think was the default for a year or two. We simplified it to improve performance and compatibility (in ~2013/4)

So, perhaps the question here is how should such tweaks be distributed. The best option at present would be a plugin that contains the relevant stylesheet and, optionally, a tiddler containing a user interface for enabling/configuring the feature. Perhaps we might add a new system tag to enable that UI to be displayed at the bottom of the existing "Theme Tweaks" tab.

**Reply by kookma** [reply]
I believe vanilla and snowwhite needs an overhaul!
many things changed from the time these theme were designed! see issue with color palettes in other thread!

**Comment by pmario** [comment]
It needs to be a new theme then. 

I do like snowwhite but I don't like additional "height" in elements inside the tiddler.

**Reply by BurningTreeC** [reply]
Ok I understand that

**Reply by BurningTreeC** [reply]
for a new theme the change is too small

**Reply by pmario** [reply]
Why? I think it could be named "Snowwhite plus" and add some tweaks to snowwhite

**Reply by BurningTreeC** [reply]
Ok, then I need to brainstorm which tweaks can be added
